internal/services/admin_management_application/resource_admin_management_application_test.go

```diff
@@ -58,7 +58,7 @@ func TestUnitAdminManagementApplicationResource_Validate_Create(t *testing.T) {
 	body := fmt.Sprintf("{ \"applicationId\": \"%s\" }", client_id.String())
 
 	httpmock.RegisterResponder(http.MethodPut, url, httpmock.NewStringResponder(http.StatusOK, body))
-	httpmock.RegisterResponder(http.MethodGet, url, httpmock.NewStringResponder(http.StatusOK, body))
+	httpmock.RegisterResponder("GET", url, httpmock.NewStringResponder(http.StatusOK, body))
```

Comment:
why did this change from `http.MethodGet` to embedded constant?

Reply:
linter was complaining on not using const, but this is a good thing as we use http lib anyways